SPECS/rubygem-zip-zip/rubygem-zip-zip.spec

```diff
@@ -0,0 +1,37 @@
+%global debug_package %{nil}
+%global gemdir %(IFS=: R=($(gem env gempath)); echo ${R[${#R[@]}-1]})
+%global gem_name zip-zip
+Summary:        a simple adapter to let all your dependencies use RubyZip v1.0.0
+Name:           rubygem-zip-zip
+Version:        0.3
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages
+URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
+Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+BuildRequires:  ruby
+Requires:       rubygem-rubyzip >= 1.0.0
+
+%description
+In Gem hell migrating to RubyZip v1.0.0? Include zip-zip in your
+Gemfile and everything's coming up roses!
```

Comment:
I know we didn't write this, but can we phrase it more professionally?

Reply:
Resolved